playbook/spec/pb_kits/playbook/kits/caption_spec.rb

```diff
@@ -32,13 +32,13 @@
 
   describe "#classname" do
     it "returns namespaced class name", :aggregate_failures do
-      expect(subject.new({}).classname).to eq "pb_caption_kit_md"
-      expect(subject.new(dark: true).classname).to eq "pb_caption_kit_md dark"
-      expect(subject.new(size: "lg").classname).to eq "pb_caption_kit_lg"
-      expect(subject.new(size: "xs").classname).to eq "pb_caption_kit_xs"
-      expect(subject.new(size: "xs", variant: "link").classname).to eq "pb_caption_kit_xs_link"
-      expect(subject.new(dark: true, size: "lg").classname).to eq "pb_caption_kit_lg dark"
-      expect(subject.new(classname: "additional_class").classname).to eq "pb_caption_kit_md additional_class"
+      expect(subject.new({}).classname).to eq "pb_caption_kit_md_light"
```

Comment:
If we are changing the default for what is rendered this will break for all kits that are targeting styles manually, which sometimes Nitro devs and others resort to,  with `[*=pb_caption_kit_md]`

Reply:
We need to do a global search in nitro and ensure this won't break existing code .